airbyte-integrations/connectors/source-tmdb/source_tmdb/spec.yaml

```diff
@@ -0,0 +1,17 @@
+documentationUrl: https://docsurl.com
```

Comment:
Update this, it must be the airbyte.io doc to your connector. See other connector to get one example.

Reply:
Updated!